### PR TITLE
CNV-23474: RN for paste string in VNC console

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.13 RN

Issue: [CNV-23474](https://issues.redhat.com//browse/CNV-23474)

Link to docs preview: https://57838--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-13-release-notes.html#virt-4-13-web-new

QE review:
- [ ] QE has approved this change. - Not needed. Text copied from new content. 

Additional information:

### CODE REVIEW

**Review by mjpytlak — COMMENTED** [review]
Peer review LGTM @sabrinajess.

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -84,6 +84,7 @@ The SVVP Certification applies to:
 === Web console
 
 //CNV-23474 Release note: NEW Paste clipboard into VNC
+You can now xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-console_virt-web-console-overview[paste a string] from your client’s clipboard into the guest when using the VNC console.
```
Comment (jeana-redhat) [inline virt/virt-4-13-release-notes.adoc:87]: Should this be a bullet like its peers in other sections?
```suggestion
* You can now xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-console_virt-web-console-overview[paste a string] from your client’s clipboard into the guest when using the VNC console.
```

**Review by jeana-redhat — COMMENTED** [review]
One thing I saw before merging

File: virt/virt-4-13-release-notes.adoc (hunk 1 above)
Comment (sabrinajess) [reply virt/virt-4-13-release-notes.adoc:87]: Whoops, yup!

**Comment by sabrinajess** [comment]
@jeana-redhat Ready for re-review and merge.

**Pull request merged**